src/utils/i18nString.ts

```diff
@@ -0,0 +1,27 @@
+import i18n from "i18n-js";
+import { Translations } from "../common/i18n/translations/type";
+
+export const i18nString = <
```

Comment:
Using a verb might help to describe better what this does. E.g. `getTranslatedStringWithI18n`

Reply:
I agree with this, have refactored it.

Reply:
just noticed this, since we're going to be using this everywhere, can we keep the fn name short?

Reply:
After some deliberation and discussion, we have decided this will be named `i18nt`!